src/register_hall_of_fame.c

```diff
@@ -0,0 +1,3069 @@
+    RegisterHofMon *mon = &data->mons[monIdx];
+
+    RegisterHallOfFame_ReplaceSpritePltt(mon->plttBak, dest, 0x20);
+}
+
+static void RegisterHallOfFame_IndivMonsScene_LoadMonOverworldSprite(RegisterHallOfFameData *data, RegisterHofMon *mon) {
+    if (mon->tsure_param[1]) {
+        SpriteRenderer_LoadCharResObjFromOpenNarc(
+            data->spriteRenderer,
+            data->spriteGfxHandler,
+            data->narcA094,
+            12,
+            TRUE,
+            NNS_G2D_VRAM_TYPE_2DMAIN,
+            55518
+        );
+        SpriteRenderer_LoadCellResObjFromOpenNarc(
+            data->spriteRenderer,
+            data->spriteGfxHandler,
+            data->narcA094,
+            13,
+            TRUE,
+            55514
+        );
+        SpriteRenderer_LoadAnimResObjFromOpenNarc(
+            data->spriteRenderer,
+            data->spriteGfxHandler,
+            data->narcA094,
+            14,
+            TRUE,
+            55514
+        );
+    } else {
+        SpriteRenderer_LoadCharResObjFromOpenNarc(
+            data->spriteRenderer,
+            data->spriteGfxHandler,
+            data->narcA094,
+            9,
+            TRUE,
+            NNS_G2D_VRAM_TYPE_2DMAIN,
+            55518
+        );
+        SpriteRenderer_LoadCellResObjFromOpenNarc(
+            data->spriteRenderer,
+            data->spriteGfxHandler,
+            data->narcA094,
+            10,
+            TRUE,
+            55514
+        );
+        SpriteRenderer_LoadAnimResObjFromOpenNarc(
+            data->spriteRenderer,
+            data->spriteGfxHandler,
+            data->narcA094,
+            11,
+            TRUE,
+            55514
+        );
+    }
+    SpriteRenderer_LoadPlttResObjFromOpenNarc(
+        data->spriteRenderer,
+        data->spriteGfxHandler,
+        data->narcA094,
+        15,
+        FALSE,
+        1,
+        NNS_G2D_VRAM_TYPE_2DMAIN,
+        55518
+    );
+    SpriteRenderer_LoadPlttResObjFromOpenNarc(
+        data->spriteRenderer,
+        data->spriteGfxHandler,
+        data->narcA094,
+        15,
+        FALSE,
+        1,
+        NNS_G2D_VRAM_TYPE_2DMAIN,
+        55519
+    );
+}
+
+static void RegisterHallOfFame_IndivMonsScene_UnloadMonOverworldSprite(RegisterHallOfFameData *data) {
+    SpriteGfxHandler_UnloadPlttObjById(data->spriteGfxHandler, 55518);
+    SpriteGfxHandler_UnloadPlttObjById(data->spriteGfxHandler, 55519);
+    SpriteGfxHandler_UnloadCharObjById(data->spriteGfxHandler, 55518);
+    SpriteGfxHandler_UnloadCellObjById(data->spriteGfxHandler, 55514);
+    SpriteGfxHandler_UnloadAnimObjById(data->spriteGfxHandler, 55514);
+}
+
+static void RegisterHallOfFame_IndivMonsScene_SetMon3dSpriteTex(RegisterHallOfFameData *data, RegisterHofMon *hofMon, int picIdx) {
+    NNSG3dResTex *resTex;
+    void *fileData;
+    const void *sp18;
+    u32 sp14 = NNS_G2dGetImageLocation(Sprite_GetImageProxy(data->monPics[picIdx]->sprite), NNS_G2D_VRAM_TYPE_2DMAIN);
```

Comment:
rename sp vars to other unowns

Reply:
i don't think `unown_A` and `unown_QuestionMark` are better names